src/Mautic/Config.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Escopecz\MauticFormSubmit\Mautic;
+
+/**
+ * Mautic Configuration
```

Comment:
Remove this comment. It's not a Mautic Configuration but rather general configuration of the form submission transfer.